README.rst

```diff
@@ -33,7 +33,7 @@ Requirements
 ------------
 
 * Redis.
-* Python 2.6, 2.7 or 3.3.
+* Python 2.7 or 3.3+.
```

Comment:
The build failed in 2.6 (https://travis-ci.org/samueldg/snappass/jobs/151697518), and it doesn't seem related to the changes I made. It seems like currently 2.6 doesn't quite work, so I thought it was more accurate to remove that in the docs/pypi classifiers.

What I suggest: I'll have a look at why this is failing. If it's an easy fix I'll incorporate that in the current PR, otherwise we could just open an issue to fix the python 2.6 support, and add it back to docs.